SPECS/rpm/rpm.spec

```diff
@@ -10,30 +10,33 @@ URL:            https://rpm.org
 Source0:        https://github.com/rpm-software-management/rpm/archive/%{name}-%{version}-release.tar.gz
 Source1:        brp-strip-debug-symbols
```

Comment:
I'm not too familiar with the origin of the brp scripts in Source1 and Source2- can you confirm that these don't have any updates that need to be made to them?

Reply:
These scripts seem good for now. The upstream source I found had an additional filter, although it did not look critical. I think we are good unless we get asked to change this.